task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -34,19 +37,22 @@ spec:
         #!/usr/bin/env bash
         source /utils.sh
         ### Try to extract binaries with configs > check binaries functionality > check opm validate ###
-        if [ ! -f ../sanity-inspect-image/image_inspect.json ] || [ -z $(cat ../sanity-inspect-image/image_inspect.json) ]; then
+
+        if [[ ! -f ../sanity-inspect-image/image_inspect.json ]] || [[ -z $(cat ../sanity-inspect-image/image_inspect.json) ]]; then
```

Comment:
See https://github.com/redhat-appstudio/build-definitions/pull/326#discussion_r1115431258, they are similar

Reply:
uff, sorry I haven't noticed :D so change it then

Reply:
@jsztuka so can we apply the same fix here?

Reply:
We do, updated.